components/eventCard/MenuDropdown.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react"
+import { Alert, StyleSheet, Text } from "react-native"
+import {
+  Menu,
+  MenuOptions,
+  MenuOption,
+  MenuTrigger
+} from "react-native-popup-menu"
+import { TouchableOpacity } from "react-native-gesture-handler"
+import { MaterialIcon } from "@components/common/Icons"
+
+interface Props {
+  isEventHost: boolean
+}
+
+const MenuDropdown = ({ isEventHost }: Props) => {
```

Comment:
I think this could use a more specific name related to events, but thanks for this.